guides/common/modules/con_prerequisites-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,45 @@
+[id="prerequisites-for-{project-context}-on-aws"]
+= Prerequisites for {Project} on AWS
+
+Installing and running {ProjectServer} and {SmartProxyServers} on Amazon Web Services (AWS) has additional requirements to your environment.
+
+* Ensure that the Amazon EC2 instance type meets or exceeds the {InstallingServerDocURL}system-requirements_{project-context}[System requirements] in _{InstallingServerDocTitle}_.
+For the best performance, use an https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html[AWS storage optimized instance].
+ifndef::foreman-deb[]
+* Use {InstallingServerDocURL}satellite-storage-requirements_{project-context}[Storage requirements] in _{InstallingServerDocTitle}_ to understand and assign the correct storage to your AWS EBS volumes.
```

Comment:
Link doesn't lead to the Storage requirements section.

Reply:
Is there a way to link to the Install prerequisites only once, instead of twice (System reqs and Storage reqs)? It would be easier to maintain one high-level link than two deep-level links.